src/algorithm.h

```diff
@@ -0,0 +1,90 @@
+#pragma once
+// Pragma once is supported on all modern compilers
+// TODO: decide whether to use include guards anyway
+
+#include <algorithm>
```

Comment:
Sure, no problem; best to avoid potential confusion (though no one should be using <algorithm.h> anyway).